proto/subscription/subscription.proto

```
@@ -0,0 +1,56 @@
+syntax = "proto3";
```

Buf (1.55.1)

Files must have a package defined.

(PACKAGE_DEFINED)